e2e/manifests/overlays/daemonset-lvmd/scheduler.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: apps/v1
```

Comment:
This is the same as daemonset manifest, so don't copy but refer daemonset manifest from kustomization.yaml.